quizer_game/templates/quizer_game/index.html

```diff
@@ -1,10 +1,115 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <title>Home</title>
+    {% load static %}
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <link rel="stylesheet" href="{% static 'quizer_game/styles/css/animate.css' %}">
+    <link rel="stylesheet" href="{% static 'quizer_game/styles/css/owl.carousel.min.css' %}">
+    <link rel="stylesheet" href="{% static 'quizer_game/fonts/ionicons/css/ionicons.min.css' %}">
+    <link rel="stylesheet" href="{% static 'quizer_game/styles/css/style.css' %}">
+    <link rel="stylesheet" href="{% static 'quizer_game/styles/css/main.css' %}">
+  </head>
+  <body id="page-top">
+    
+    <header id="masthead">
+      <!-- login version -->
+      <!-- <div class="email">
+        <a href="">Email: hello55@gmail.com</a>
+      </div>
+      <div class="top">
+        <a class="create" href="">Create quiz</a>
+        <a class="history" href="">History</a>
+        <a class="logout" href="">Logout</a>
+      </div> -->
+
+      <div class="top">
+        <a href="/quizer/login/" class="signup">Sign Up</a>
+        <a href="/quizer/login/" class="login">Login</a>
+      </div>
```

Comment:
Shouldn't use hard code url. Please consider fixing it.

Reply:
I've already improve it. Please review again.